English_transcripts/school/Epic Quest/sharedAssets/C1_english_paint_pots_complete.txt

```diff
@@ -0,0 +1 @@
+You painted all the pots. Great job.
```

Comment:
This is the same as the one two steps above. Is this needed?

Reply:
Yes they are two different audios.